build.gradle

```diff
@@ -89,15 +89,28 @@ idea {
     }
 }
 
+project.version = '0.7.0'
 ext.versionSource = versionSource
-ext.globalVersion = "0.7.0"
+ext.globalVersion = project.version
 
 // To set fishlibVersion, go to project-local gradle.properties file
 // (or, use the gradle.properties in $USER_HOME/.gradle for a global override)
 // If you have fishlib open in IDE, also set includeFish=true to get the
 // intellij projects wired up across repositories.
 
-Set<Project> modsAreBasic = subprojects.findAll {it.name in ['deephaven-wheel', 'deephaven2-wheel', 'envoy', 'grpc-proxy', 'web-client-ui', 'protoc', 'pyclient', 'sphinx', 'docker-java-and-python', 'docker-runtime-base'] }
+Set<Project> modsAreBasic = subprojects.findAll {it.name in [
+        'deephaven-wheel',
+        'deephaven2-wheel',
+        'envoy',
+        'grpc-proxy',
+        'web-client-ui',
+        'protoc',
+        'pyclient',
+        'sphinx',
+        'docker-java-and-python',
+        'docker-runtime-base',
+        'docker-server',
+        'docker-server-pytorch'] }
```

Comment:
newline before `] }` (and possibly trailing comma) so it is cleaner to add new projects?